flood_adapt/object_model/direct_impacts.py

```diff
@@ -86,13 +91,34 @@ def set_hazard(self, scenario: ScenarioModel, database_input_path: Path) -> None
         """
         self.hazard = Hazard(scenario, database_input_path)
 
+    def set_hazard_fiat(self, fiat_adapter: FiatAdapter):
+        map_fn = self.get_sfincs_map_path(self.hazard.sfincs_map_path)
+
+        is_risk = self.hazard.event_mode == Mode.risk
+        var = "zsmax" if self.hazard.event_mode == Mode.risk else "risk_maps"
+
+        fiat_adapter.fiat_model.setup_hazard(
+            map_fn=map_fn,
+            map_type="water_depth",
+            rp=None,  # change this in new version
+            crs=None,  # change this in new version
+            nodata=-999,  # change this in new version
+            var=var,
+            chunks="auto",
+            name_catalog=None,
+            risk_output=is_risk,
+        )
```

Comment:
This function should be in the FiatAdapter class (there is already an empty method for this purpose there).
And the map_type should be read by the site.toml as we discussed.

Reply:
good point! I see now what you mean. You mean to move this part to fiat integrator. Done